### PR TITLE
V1.4: Documentation

### PR DESCRIPTION
**Pull request by filbertphang**
Closes:

- [x] #187 
- [x] #188 
- [x] #199

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -567,21 +567,21 @@ testers are expected to do more *exploratory* testing.
...
 1. **Viewing recipes** <br> _(Preconditions: User is on the default MainWindow page.)_
    1. Click on a Recipe Card once.
-   2. Press "P" on the keyboard.
+   2. Press the "P" key.
     <br>**Expected:** A modal showing containing the name, duration, portion, ingredients, steps, and tags fields of the recipe appears.  
 
 2. **Edit form** <br> _(Preconditions: User is on the default Main Window page.)_
     1. Click on a recipe card to select it.
-    2. Press the "F" key on the keyboard.
+    2. Press the "F" key.
```
Comment (SeeuSim) [inline docs/DeveloperGuide.md:607]: <kbd>F</kbd>

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -567,21 +567,21 @@ testers are expected to do more *exploratory* testing.
...
        <br>**Expected:** A form modal appears, containing text input fields for name, duration, portion, ingredients, steps, and tags, prepopulated accurate to the recipe data, as well as a `Cancel` and `Save` button on the bottom right.
     3. Edit any of the fields in the form, using the format provided in the user guide as reference.
-    4. Click "Save" at the bottom of the form to save and exit the form.
+    4. Click `Save` at the bottom of the form to save and exit the form.
```
Comment (SeeuSim) [inline docs/DeveloperGuide.md:610]: <kbd>Save</kbd>

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -591,9 +591,21 @@ testers are expected to do more *exploratory* testing.
    1. Type `addf` in the command box and enter.
     <br>**Expected:** A form modal appears, containing empty text input fields for name, duration, portion, ingredients, steps, and tags, as well as a `Cancel` and `Save` button on the bottom right.
```
Comment (SeeuSim) [inline docs/DeveloperGuide.md:618]:
```md
<kbd>Cancel</kbd>
```

Comment (ZHTang29) [reply docs/DeveloperGuide.md:618]: Will add these keyboard shortcut formatting after current changes to UG are merged.

**Review by ZHTang29 — APPROVED** [review]
Good for merge

**Pull request merged**